app/javascript/components/venue/VenueNew.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import Axios from 'axios';
+
+export default class VenueNew extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.state = { venue: { address: "", decription: "", people_capacity: "" } };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(venue) {
+    this.setState({
+      [venue.target.name]: venue.target.value
+    });
+  }
+
+  handleSubmit(venue) {
+    const { address, description, people_capacity } = this.state;
+
+    Axios
```

Comment:
add csrf token to header like in VenueEdit

Reply:
done